keps/sig-scheduling/4671-gang-scheduling/README.md

```diff
@@ -458,6 +458,39 @@ not be split into two. A `LeaderWorkerSet` is a good example of it, where a sing
 of a single leader and `N` workers and that forms a scheduling (and runtime unit), but workload as a whole
 may consist of a number of such replicas.
 
+#### Basic Policy Extension
+
+While Gang Scheduling focuses on atomic, all-or-nothing scheduling, there is a significant class
+of batch workloads that requires best-effort optimization without
```

Comment:
Let's not mention batch here - we want the "Workload" to represent arbitrary workloads and clearly not workload doesn't consist of gangs.